src/common/credentials.ts

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+import MANAGED_PATHS from './managed-paths';
+
+let keytar: any;
+try {
+  keytar = require('keytar');
+} catch {
+  keytar = null;
+}
+const config_path = path.join(os.homedir(), MANAGED_PATHS.HIDDEN, 'config.json');
+
+const getConfig = async () => {
+  await fs.ensureFile(config_path);
+  const config = await fs.readJSON(config_path).catch(() => ({}));
```

Comment:
try/catch block for `await` instead of appending `catch()`?

Reply:
meh if it was a more interesting catch I would, but not worth adding the extra lines imo